### DISCUSSION TITLE
Students using Copilot - "rate limit exceeded" error

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#44097] [GitHub Education]
Hi y'all 👋 

This week, we've had to take down some of our Copilot resources for maintenance. While we did our best to source extra capacity elsewhere, we've been forced to limit some free users for the period of time where one of our busiest models is out of commission.

We expect that this will cease to impact your use of Copilot after the 16th of January. In the meantime, you will have less frequent suggestions, but you should still be getting suggestions - just with rate limits in between. You may see these errors specifically if you look in the output pane or logs for your IDE.

We are also aware that some students are still experiencing issues with redeeming and accessing Copilot and that the team are currently working to resolve this. We'll update the community when we have more information on that incident in a separate discussion.

Thanks for your patience and apologies for the inconvenience caused.

**Comment by NeisserMS** [comment]
Es cierto, tuve problemas para obtener el copilot, pero al fin ya lo tengo y estoy fascinado, va de maravilla.
